tools/migration/main.go

```diff
@@ -70,36 +82,56 @@ func main() { // nolint: deadcode
 		if err != nil {
 			exitErr(err.Error())
 		}
+
 		logger := internal.NewLogger(logFile, getVerbose())
```

Comment:
I don't see us closing the logfile anywhere with logger.Close().  We should do that.  It's actually a pain to do because I don't think we can use defers when we exit.

Feel free to open up a follow on issue if you want to handle this work after this PR.

Reply:
I'll file a PR -- I'd like to discuss the best way, because logger.Close() returns an error, also we could pass the logger to exitErr, etc etc.

Reply:
#2760